### PR TITLE
Refactor reproterms

### PR DESCRIPTION
**Pull request by sanuann**

### CODE REVIEW

File: terms/ResponseOption [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"@context": {
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#"
+    },
+    "@id": "ResponseOptions",
```
Comment (satra) [inline terms/ResponseOption:6]: id and term filename are different. and i thought ResponseOption is a schema.

File: terms/variableName [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+    "@type": "rdf:Property",
+    "rdfs:comment": "The name used to represent an item.",
+    "rdfs:label": "variableName",
+    "schema:domainIncludes": {
+        "@id": "reproschema:ScoringLogic"
+    },
+    "schema:rangeIncludes": {
+        "@id": "schema:Text"
+    },
+    "schema:sameAs": {
```
Comment (satra) [inline terms/variableName:19]: isn't this just pointing to itself?

File: terms/valueType [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "schema:rangeIncludes": [
+        {
+            "@id": "schema:Text"
+        },
+        {
+            "@id": "rdf:langString"
+        }
+    ],
+    "schema:sameAs": {
```
Comment (satra) [inline terms/valueType:24]: pointing to itself.

File: terms/ResponseOption (hunk 1 above)
Comment (sanuann) [reply terms/ResponseOption:6]: sorry, this one i committed by mistake. it is a schema. I'll delete this file.

File: terms/variableName (hunk 2 above)
Comment (sanuann) [reply terms/variableName:19]: yes. that's how I had seen it in some of the nodes in `schema.org`. But I don't see it anymore in their latest schema version. I didn't quite understand it to be honest. I'll remove it then?

Comment (sanuann) [reply terms/variableName]: deleted it.

File: terms/allow [hunk 4]
```diff
@@ -16,7 +17,6 @@
         {
             "@id": "reproschema:Activity"
         }
-
     ],
     "schema:rangeIncludes": {
         "@id": "schema:Text"
```
Comment (sanuann) [inline terms/allow:22]: @satra can you check the range here?

Comment (satra) [reply terms/allow:22]: i think range would be `schema:Thing` since all the possibilities are terms that are classes.

Comment (sanuann) [reply terms/allow:22]: yes true. thanks!

**Pull request merged**